pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -39,6 +39,14 @@ const (
 // for system bootstrapping.  This method takes additional users and groups that should be added
 // to the strategies.  Use GetBoostrapSCCAccess to produce the default set of mappings.
 func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string][]string, sccNameToAdditionalUsers map[string][]string) []kapi.SecurityContextConstraints {
+	// define priorities here and reference them below so it is easy to see, at a glance
+	// what we're setting
+	var (
+		// this is set to 10 to allow wiggle room for admins to set other priorities without
+		// having to adjust anyUID.
+		securityContextConstraintsAnyUIDPriority = 10
```

Comment:
parse exception.  Do you want the lower score (except 0) to be considered higher priority?